orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -103,42 +104,57 @@ public TypeDescription type() {
   private ORCSchemaUtil() {
   }
 
-  public static TypeDescription convert(Schema schema) {
+  public static TypeDescription convertSchema2TypeDesc(Schema schema) {
+    return convertSchema2TypeDescWithLength(schema).first();
+  }
+
+  public static Pair<TypeDescription, Integer> convertSchema2TypeDescWithLength(Schema schema) {
```

Comment:
Based on the discussion of the mailing list, I tried to estimate the average length of the field here. @openinx

Reply:
Why not just customize a `OrcSchemaVisitor` to calculate the estimated avg row size  to decouple with the schema  convertion logic ?

Reply:
Besides, we also need a unit tests to address correctness of this estimate approach.